diffkemp/simpll/DifferentialFunctionComparator.cpp

```diff
@@ -350,6 +364,50 @@ int DifferentialFunctionComparator::cmpBasicBlocks(const BasicBlock *BBL,
     return 0;
 }
 
+std::vector<SyntaxDifference> DifferentialFunctionComparator::findAsmDifference(
+        const Value *L, const Value *R, const Function *ParentL,
+        const Function *ParentR) const {
+    auto FunL = dyn_cast<Function>(L);
+    auto FunR = dyn_cast<Function>(R);
+
+    if (!FunL || !FunR)
+        // Both values have to be functions
+        return {};
+
+    if (!FunL->getName().startswith("simpll__inlineasm") ||
```

Comment:
Define a macro for the "simpll_inlineasm" string so that it can be changed at a single place. I'd define it in passes/FunctionAbstractionsGenerator.h.